games-emulation/es-de/es_de-3.1.0.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="Gaming frontend for your multi-platform collection"
+DESCRIPTION="ES-DE (EmulationStation Desktop Edition) is a frontend for browsing and launching games from your \
+multi-platform collection. It comes preconfigured for use with a large selection \
+of emulators and game engines."
+HOMEPAGE="https://es-de.org"
+COPYRIGHT="2024 Northwestern Software AB"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="842c9966eb73efb3436a4df2dfdd66063ce7361c"
+SOURCE_URI="https://gitlab.com/es-de/emulationstation-de/-/archive/$srcGitRev/emulationstation-de-$srcGitRev.tar.gz"
+CHECKSUM_SHA256="52c820beddba7e08014f589120c087d6b84b79a88b6213bf13a4e025fd728285"
+SOURCE_FILENAME="emulationstation-de-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="emulationstation-de-$srcGitRev"
+ADDITIONAL_FILES="es-de.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+        es_de = $portVersion
+        app:es_de = $portVersion
+        "
+
+REQUIRES="
+        haiku$secondaryArchSuffix
+        lib:libGL$secondaryArchSuffix
+        lib:libavcodec$secondaryArchSuffix
+        lib:libavformat$secondaryArchSuffix
+        lib:libavutil$secondaryArchSuffix
+        lib:libcurl$secondaryArchSuffix
+        lib:libfreeimage$secondaryArchSuffix
+        lib:libfreetype$secondaryArchSuffix
+        lib:libharfbuzz$secondaryArchSuffix
+        lib:libgit2$secondaryArchSuffix >= 1.5.0
```

Comment:
for REQUIRES, the version constraints are filled automatically by haikuporter, so this isn't needed here.

Reply:
Ok, I've removed it now!

Reply:
Good to go now?